package.json

```diff
@@ -28,10 +28,10 @@
     "redux": "^3.3.1"
   },
   "peerDependencies": {
-    "react": "^0.14.7 || ^15.0.0-rc.1"
+    "react": "^0.14.7 || ^15.0.1"
   },
   "devDependencies": {
-    "autoprefixer-loader": "^3.2.0",
+    "postcss-loader": "^0.8.2",
```

Comment:
Maybe I'm missing something, but shouldn't we also update Webpack configs? Where is `postcss-loader` used?